plugins/00_dokku-standard/install

```diff
@@ -33,7 +35,7 @@ After=docker.target
 [Service]
 Type=simple
 User=dokku
-ExecStart=/usr/local/bin/dokku ps:restore
+ExecStart=$dokku_path ps:restore
```

Comment:
@michaelshobbs @josegonzalez I added the changes to this file to fix the restore process on ArchLinux. It works there just fine and should also work without issues on all other platforms.